.github/workflows/master.yml

```diff
@@ -0,0 +1,145 @@
+name: master
+
+on:
+  push:
+    branches: 
+      - master
+  pull_request:
+    branches: 
+      - master
+
+jobs:
+  python3_5:
+    runs-on: ubuntu-latest
+    env:
+      SPARK_VERSION: 2.3.4
+      PANDAS_VERSION: 0.23.4
+      PYARROW_VERSION: 0.10.0
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+
+    - name: setup_python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.5
+
+    - name: before_install
+      run: ./dev/download_travis_dependencies.sh
+
+    - name: install
+      run: |
+        sudo apt-get update
+        pip install --upgrade cython
+        pip install -r requirements-dev.txt
+        pip install pandas==$PANDAS_VERSION pyarrow==$PYARROW_VERSION
+        pip install 'pyspark>=2.4'
+        pip list
+
+    - name: before_script
+      run: |
+        export SPARK_HOME="$HOME/.cache/spark-versions/spark-$SPARK_VERSION-bin-hadoop2.7"
+        export QT_QPA_PLATFORM="offscreen"
+        ./dev/lint-python
+
+    - name: script
+      run: ./dev/pytest
+
+    - name: after_success
+      run: bash <(curl -s https://codecov.io/bash)
+
+  python3_6:
+    runs-on: ubuntu-latest
+    env:
+      SPARK_VERSION: 2.4.4
+      PANDAS_VERSION: 0.24.2
+      PYARROW_VERSION: 0.13.0
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+
+    - name: setup_python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.6
+  
+    - name: before_install
+      run: ./dev/download_travis_dependencies.sh
+  
+    - name: install
+      run: |
+        sudo apt-get update
+        pip install --upgrade cython
```

Comment:
Can you test and setup env via using `conda` for Python 3.6 and 3.7? We should test Conda as well, which is the preferred installation way.